packages/server/src/tests/epics/epicProfileWithImageTest_us.ts

```diff
@@ -0,0 +1,92 @@
+                'Unlike many others, the Guardian has no shareholders and no billionaire owner. Just the determination and passion to deliver high-impact global reporting, always free from commercial or political influence. Reporting like this is vital to establish the facts: who is lying and who is telling the truth.',
+                'And we provide all this for free, for everyone to read. We do this because we believe in information equality. Greater numbers of people can keep track of the global events shaping our world, understand their impact on people and communities, and become inspired to take meaningful action. Millions can benefit from open access to quality, truthful news, regardless of their ability to pay for it.',
+                'If there were ever a time to join us, it is now. Every contribution, however big or small, powers our journalism and sustains our future.',
+            ],
+            highlightedText:
+                'Support the Guardian from as little as %%CURRENCY_SYMBOL%%1 – it only takes a minute. Thank you.',
+            cta: CTA,
+            secondaryCta: { type: SecondaryCtaType.ContributionsReminder },
+            separateArticleCount: { type: 'above' },
+            showChoiceCards: true,
+        },
+        {
+            name: 'V1_authored_plain',
+            modulePathBuilder: epic.endpointPathBuilder,
+            paragraphs: [
+                "I write from Ukraine, where I've spent much of the past six months, reporting on the build-up to the conflict and the grim reality of war. It has been the most intense time of my 30-year career. In December I visited the trenches outside Donetsk with the Ukrainian army; in January I went to Mariupol and drove along the coast to Crimea; on 24 February I was with other colleagues in the Ukrainian capital as the first Russian bombs fell.",
+                "This is the biggest war in Europe since 1945. It is, for Ukrainians, an existential struggle against a new but familiar Russian imperialism. Our team of reporters and editors intend to cover this war for as long as it lasts, however expensive that may prove to be. We are committed to telling the human stories of those caught up in war, as well as the international dimension. But we can't do this without the support of Guardian readers. It is your passion, engagement and financial contributions which underpin our independent journalism and make it possible for us to report from places like Ukraine.",
+                'If you are able to help with a monthly or single contribution it will boost our resources and enhance our ability to report the truth about what is happening in this terrible conflict.',
+                'Thank you.',
+                '<strong>Luke Harding</strong>, Foreign correspondent',
+            ],
+            highlightedText: '',
```

Comment:
`highlightedText` is optional so you can just leave it out here

Reply:
Done